QKSMS/src/main/java/com/moez/QKSMS/MmsConfig.java

```diff
@@ -329,7 +329,7 @@ private static void loadMmsSettings(Context context) {
             while (true) {
                 nextElement(parser);
                 String tag = parser.getName();
-                if (tag == null) {
+                if (tag == null || tag.isEmpty()) {
```

Comment:
Running this code was failing because it was somehow getting an empty tag. This *may* not be necessary anymore, but it seems like a fair change regardless.